packages/cloud_firestore/pubspec.yaml

```diff
@@ -14,9 +14,9 @@ flutter:
 dependencies:
   flutter:
     sdk: flutter
-  firebase_core: "^0.2.2"
   meta: "^1.0.5"
   collection: "^1.14.3"
+  firebase_core: "^0.2.5+1"
```

Comment:
Shouldn't this be in dev_dependencies?

Reply:
Since the plugin explicitly uses `firebase_core`, it has to be in `dependencies`